study.md

```diff
@@ -57,7 +59,7 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+.create will make new objects, but using .new will create new instances of the object
```

Comment:
`#new` instantiates a new object but doesn't save it to the database.  `#save` needs to be invoked on it afterward to save it.